<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Mango</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>991</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVideoWidget" name="viewfinder" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionBegin_camera_capture"/>
    <addaction name="actionEnd_camera_capture"/>
    <addaction name="actionDigiCam_report"/>
   </widget>
   <widget class="QMenu" name="menuThrusters">
    <property name="title">
     <string>Thrusters</string>
    </property>
    <addaction name="actionDisplay_thruster_setup_dialog"/>
   </widget>
   <widget class="QMenu" name="menuJoystick">
    <property name="title">
     <string>Joystick</string>
    </property>
    <addaction name="actionDisplay_joystick_setup_dialog"/>
   </widget>
   <widget class="QMenu" name="menuData_graphs">
    <property name="title">
     <string>Data graphs</string>
    </property>
    <addaction name="actionShow_graphs"/>
    <addaction name="actionHide_graphs"/>
   </widget>
   <widget class="QMenu" name="menuAdditional_functions">
    <property name="title">
     <string>Additional functions</string>
    </property>
    <addaction name="actionPump_off_on"/>
    <addaction name="actionLight_on_off"/>
    <addaction name="actionRecalibrate_IMU"/>
   </widget>
   <widget class="QMenu" name="menuRegulators">
    <property name="title">
     <string>Regulators</string>
    </property>
    <widget class="QMenu" name="menuDepthReg">
     <property name="title">
      <string>Depth</string>
     </property>
     <addaction name="actionDepthOff"/>
     <addaction name="actionDepthOn"/>
     <addaction name="actionDepthOnWC"/>
    </widget>
    <widget class="QMenu" name="menuYawReg">
     <property name="title">
      <string>Yaw</string>
     </property>
     <addaction name="actionYawOff"/>
     <addaction name="actionYawOn"/>
     <addaction name="actionYawOnWC"/>
    </widget>
    <widget class="QMenu" name="menuRollReg">
     <property name="title">
      <string>Roll</string>
     </property>
     <addaction name="actionRollOff"/>
     <addaction name="actionRollOn"/>
     <addaction name="actionRollOnWC"/>
    </widget>
    <widget class="QMenu" name="menuPitchReg">
     <property name="title">
      <string>Pitch</string>
     </property>
     <addaction name="actionPitchOff"/>
     <addaction name="actionPitchOn"/>
     <addaction name="actionPitchOnWC"/>
    </widget>
    <addaction name="menuDepthReg"/>
    <addaction name="menuYawReg"/>
    <addaction name="menuRollReg"/>
    <addaction name="menuPitchReg"/>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuThrusters"/>
   <addaction name="menuJoystick"/>
   <addaction name="menuData_graphs"/>
   <addaction name="menuAdditional_functions"/>
   <addaction name="menuRegulators"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="telemetryDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Telemetry</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="telemetryDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="12" column="1">
      <widget class="QLabel" name="teleCurrentLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="teleDepthLabelD">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Depth:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="asfLabelD">
       <property name="text">
        <string>ASF:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="teleVoltageLabelD">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Voltage:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="asfLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="teleVersionLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="teleVersionLabelD">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="teleTempLabelD">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Twater:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="teleDepthLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="teleCamSelLabelD">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>CamSel:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="teleVoltageLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="teleTempLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="teleCamSelLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="teleCurrentLabelD">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Current:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="regulatorsDockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>188</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Regulators</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="regulatorsDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yawSpinBox">
       <property name="suffix">
        <string notr="true">°</string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="yawRegulatorCB">
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="depthRegulatorCB">
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="pitchSpinBox">
       <property name="suffix">
        <string notr="true">°</string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="depthSpinBox">
       <property name="suffix">
        <string>m</string>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="rollSpinBox">
       <property name="suffix">
        <string notr="true">°</string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="rollRegulatorCB">
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="pitchRegulatorCB">
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="compassWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>500</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Compass</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="compassDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="compassLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionBegin_camera_capture">
   <property name="text">
    <string>Begin camera capture</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionEnd_camera_capture">
   <property name="text">
    <string>End camera capture</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionDisplay_thruster_setup_dialog">
   <property name="text">
    <string>Display thruster setup dialog</string>
   </property>
  </action>
  <action name="actionDisplay_joystick_setup_dialog">
   <property name="text">
    <string>Display joystick setup dialog[WIP]</string>
   </property>
  </action>
  <action name="actionShow_graphs">
   <property name="text">
    <string>Show graphs</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionHide_graphs">
   <property name="text">
    <string>Hide graphs</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionLight_off_on">
   <property name="text">
    <string>Light off/on</string>
   </property>
  </action>
  <action name="actionPump_off_on">
   <property name="text">
    <string>Pump on/off</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionDepthOff">
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionDepthOn">
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionDepthOnWC">
   <property name="text">
    <string>On with current value</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionYawOff">
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string>Shift+Y</string>
   </property>
  </action>
  <action name="actionYawOn">
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string>Y</string>
   </property>
  </action>
  <action name="actionYawOnWC">
   <property name="text">
    <string>On with current</string>
   </property>
   <property name="shortcut">
    <string>Alt+Y</string>
   </property>
  </action>
  <action name="actionRollOff">
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionRollOn">
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionRollOnWC">
   <property name="text">
    <string>On with current</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionPitchOff">
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionPitchOn">
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionPitchOnWC">
   <property name="text">
    <string>On with current</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionLight_on_off">
   <property name="text">
    <string>Light on/off</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionDigiCam_report">
   <property name="text">
    <string>DigiCam report</string>
   </property>
  </action>
  <action name="actionRecalibrate_IMU">
   <property name="text">
    <string>Recalibrate IMU</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>depthRegulatorCB</tabstop>
  <tabstop>depthSpinBox</tabstop>
  <tabstop>yawRegulatorCB</tabstop>
  <tabstop>yawSpinBox</tabstop>
  <tabstop>rollRegulatorCB</tabstop>
  <tabstop>rollSpinBox</tabstop>
  <tabstop>pitchRegulatorCB</tabstop>
  <tabstop>pitchSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
